### ISSUE TITLE
bam2cns unexpected return 

### ISSUE THREAD

**Issue by stanleyouth** [BioInf-Wuerzburg/proovread#10]
Dear development team:

I tested proovread using sample data according to the README.

command line is:
bin/proovread --sample --pre pr-sample

details about the error:
perl -I/mnt/md1200f/zengyan/soft/soft/soft/proovread/bin/../lib/ /mnt/md1200f/z\
engyan/soft/soft/soft/proovread/bin/bam2cns --append 1 --bam pr-sample/bwa-sr-1\
/pr-sample.bam --bin-size 20 --coverage 11.25 --ignore-weak-reads 0 --mask-weak\
-reads 0 --max-ins-length 0 --max-ref-seqs 100 --prefix pr-sample/bwa-sr-1/pr-s\
ample.0 --qv-offset 33 --ref pr-sample/read-long/pr-sample.fq --ref-offset 0
#------------------------------------------------------------------------------#
[Wed Jul 29 17:06:23 2015] Batch 2/2
perl -I/mnt/md1200f/zengyan/soft/soft/soft/proovread/bin/../lib/ /mnt/md1200f/z\
engyan/soft/soft/soft/proovread/bin/bam2cns --append 1 --bam pr-sample/bwa-sr-1\
/pr-sample.bam --bin-size 20 --coverage 11.25 --ignore-weak-reads 0 --mask-weak\
-reads 0 --max-ins-length 0 --max-ref-seqs 100 --prefix pr-sample/bwa-sr-1/pr-s\
ample.1 --qv-offset 33 --ref pr-sample/read-long/pr-sample.fq --ref-offset 2128\
03
#------------------------------------------------------------------------------#
Use of uninitialized value in array element at /mnt/md1200f/zengyan/soft/soft/s\
oft/proovread/bin/../lib/Sam/Seq.pm line 441.
#------------------------------------------------------------------------------#
[Wed Jul 29 17:06:23 2015] bam2cns unexpectedly returned
        Exited at 'main'
        'bin/proovread', line 1605
        Last call 'main::correct_sr_mt'
        'bin/proovread', line 865

I executed both bam2cns command lines again, they returned errors like this:
Use of uninitialized value in array element at /mnt/md1200f/zengyan/soft/soft/soft/proovread/bin/../lib/Sam/Seq.pm line 441.
Use of uninitialized value in array element at /mnt/md1200f/zengyan/soft/soft/soft/proovread/bin/../lib/Sam/Seq.pm line 441, <SAM> line 133.
Use of uninitialized value in array element at /mnt/md1200f/zengyan/soft/soft/soft/proovread/bin/../lib/Sam/Seq.pm line 441, <SAM> line 127.
Use of uninitialized value in array element at /mnt/md1200f/zengyan/soft/soft/soft/proovread/bin/../lib/Sam/Seq.pm line 441, <SAM> line 122.

Could you please help me with this problem ?
Thank you!

**Comment by thackl** [comment]
Hi,

this is a rather strange error... The easiest way for me to analyse it, would be if  rerun the sample command:

```bash
proovread --pre pr-sample --sample -o -k 2>pr-sample.log
```
and send me the log and the entire pr-sample folder in an email.

Cheers

**Comment by stanleyouth** [comment]
I think I know the reason now. 
Because my server doesn't support online installation, I download proovread in windows environment using my laptop. Then I transferred proovread to the server and compiled the source code. Thus, many files are in DOS format. I checked the sample sequence files just now, they are DOS format, too. 
I set these file format back into UNIX, and then proovread runs smoothly,

Wed Jul 29 20:47:32 2015] Total time spent 00:00:38
[Wed Jul 29 20:47:32 2015] Done

Thank you for your concerning!
